### DISCUSSION TITLE
Thinking Cap #1 💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#25] [Thinking Caps]
#### Note: responses submitted after Sunday 3pm (6/30) will not be eligible for extra credit.

> ## What do you think are problems with the way we use data and GIS?

Post your thinking cap response and reply to others in this thread here. 

##### Refer to this [documentation](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week1/thinking_cap) if you need a reminder on how to complete the assignment.

**Comment by jasminereyz** [comment]
I believe there are many times when data used in GIS can be restricted or easily incorrect as it can be hard to manipulate technical software. Also, much of this software can be outdated, especially in an age where technology is evolving so rapidly, it can be hard to keep up with software as we saw with Leaflet. This also makes me wonder how accurate data can be as we do not fully know if the software is used properly or if data is collected properly. However, I think if data and softwares can be consistent, it can be very useful in the future.

**Reply by kianaptran** [reply]
This is so true! I have trouble keeping up with the constantly changing technological innovations and am always reverting to older methods. This would be a problem as there could be more efficient, effective, and accurate methods of data collecting that people would be missing out on if those individuals were not as tech-savvy. There are always errors in programming with new technology, so there's also the possibility of these bugs impacting data and GIS.

**Reply by sjeyakum** [reply]
I agree with your comment about a lot of software becoming outdated, but I also find it interesting that even some of those "outdated" programs make "comebacks" with new features to keep up with other technological advancements. Sometimes I find it easier to keep up with the advancements made within programs rather than learning completely new programs, but I understand how doing so would keep me from exploring other ways that may be more efficient (ties back to Kiana's reply). I guess this makes me wonder how different companies approach the potential learning curve when revealing new softwares, as it is important that they work with users to build trust but stay true to their program's original goals/purposes.

**Comment by kianaptran** [comment]
A problem with how we use data and GIS is that information collected can be manipulated and exploited in unintentional ways that may cause harmful outcomes. For example, information on the trends of nicotine sales can be collected and mapped, which allows nicotine marketers to target highly concentrated areas and make the people in those areas more susceptible to nicotine addictions. Additionally, a problem with data collection is how the collector organizes the information. To elaborate, depending on how data is organized, the results can be skewed and lead to misrepresentation. Many Asian American health data have this problem as all Asian groups (Southeast, South, East, etc.) are aggregated, which leads to many group health problems being overlooked.

**Reply by jasminereyz** [reply]
Kiana, I think your take on nicotine sales is critical as I myself have never even thought of this. Yet, I think this is very important to note and make known for the bettering of those vulnerable and being targeted by nicotine marketing. I just do wonder how this can be done efficiently as I am afraid that this data will just always be accessible and easily created.

**Reply by stinest** [reply]
I agree with you both!! Data can so easily be manipulated in harmful ways and it is so important to raise awareness. As for Kiana's note, have you looked into any case studies addressing data ethics? There might be some interesting solutions to this.

**Reply by pwinnie2026** [reply]
Your point on nicotine sales, as Jasmine mentioned, was really interesting as I did not initially think of using the data for those means. As for your point on Asian American health data, data disaggregation is so important and is something that I try to keep updated on personally. Context is especially important in better representing our communities. People may not report overcrowding in multigenerational, multifamily households on surveys or data collection if they view it as helping their family members/friends.

**Reply by nesslee123** [reply]
You make really great points throughout your response! I especially agree with your point regarding how results can be skewed to mislead viewers or readers because data is not always unbiased or portrayed accurately and in some cases may be used to further specific agendas. I also liked your point about how data can be used for exploitation and to harm others because this is a new perspective that I didn’t consider. Overall, I really enjoyed your response and your perspectives!

**Comment by giaari15** [comment]
The problems with the way we use data and GIS is that information and data is biased at every step of the collection and gathering process. The programs and data itself is a reflection of the society that we exist in and in turn harbors the same systemic issues we face and perpetuate. For example, if an academic institution is doing research on low-income single mothers in the Los Angeles area by survey and interview, a lot of data would never be collected by those most affected by their financial status because of access to even knowing about the study, the time sacrifice to go and complete it, and if the survey is not that extensive, the quality of the answers may by impacted to simply get it done. Furthermore, if a medical study is being completed, the participants would likely be disproportionately white because medical researchers have a history of experimenting on people of color, and especially black people, which facilitates a justified mistrust on medical research. So when people then draw conclusions based on data and GIS, if they do not approach it with a critical framework, it misrepresents "truths" because a lot of data is presented as objective when there is much more subjectivity than is typically considered.

**Reply by stinest** [reply]
I completely agree! In my own research, I have noticed similar issues with data collection in underrepresented communities. It is especially dangerous since most people see data as an inherently reliable and factual source, but by introducing bias at each step of its collection, it definitely can be anything but.

**Reply by jasminereyz** [reply]
I also agree with this! I think it can be very harmful for medical studies to be manipulated this way as people go to this data for help on their own research, creating a feedback loop of misinformation and treatment. This makes me wonder in how credible other studies using GIS can be!

**Reply by chrissanrow** [reply]
I really appreciate the example you used in describing the drawbacks of treating data as solely objective and truthful. Oftentimes, we don't consider **how** certain groups may be misrepresented, or even not represented at all in data, and challenging the underlying structures that cause this misrepresentation is key to progressing towards honest and representative data.

**Comment by stinest** [comment]
While there are many advantages to using data and GIS, like any technology, it can be misused or cause more problems than it fixes. I believe one of the largest issues we are facing is equal access. As mentioned in Crampton and Krygier's article, it can exacerbate existing social inequalities. Even as data and GIS technology becomes more developed, its distribution will often lag between social groups who either can't access the right tools or the knowledge to use it. Because these technologies are only truly effective when people have access to it, restricting this access significantly limits its capacity for discovery and progress. In using data and GIS, I agree with Crampton and Krygier's point that it is just as important to also study how technology works with society to unlock its full potential.

**Reply by giaari15** [reply]
I agree! Access is a huge issue in data and GIS especially because of the structures we exist in that continue to limit it. I remember during quarantine that internet which seems widely accessible in the present day was actually a huge issue especially with reliability. Also I feel the accessibility issue is very tied with keeping voices unheard to keep issues on the low because of this perceived objectivity of data.

**Reply by chrissanrow** [reply]
I love how you emphasized the importance of equal access to technologies. I think another potential point to consider is that although the idea of a "digital divide" suggests the need to distribute technology evenly across society, we should also prioritize ensuring that these technologies are appropriate in serving such marginalized communities. Specifically, the context of each community should be considered with these technologies, which is why empowering such oppressed groups to take control and develop these technologies for their own needs and goals is so meaningful.

**Comment by chrissanrow** [comment]
GIS and data as a whole are often treated as sterile, scientific, and objective. However, these technologies are inherently highly politicized and easily misused. Take, for instance, the WordNet database. While seemingly a purely taxonomical classification system, closer inspection reveals the penetration of human bias and prejudices into what should be purely scientific. Implicit biases can be found in the database's subclasses, such as the term "hermaphrodite" being located under the branch Person > Sensualist > Bisexual. In the case of GIS, when the development of cartographical tools is gatekept to the hands of the elite, it can often be manipulated to serve militaristic or political agendas. To be sure, in both these cases, progress can be made in diverting the production of these technological and cartographical tools to the hands of those most marginalized by them (whether it be through open-source efforts or supporting research roles).

**Reply by giaari15** [reply]
Fully. I did not even know that about the WordNet database. I feel data hinges too much on classification that does not holistically represent how individual people approach gender and sexuality. Echoing that GIS and data are tools, but to actually serve the people it has got be be by the people. It is a larger structure of viewing STEM as objective when it is just as subjective as humanities and an extension of the political. Data, GIS, and STEM overall are not inherently capitalistic but the way it has been framed and utilized has pushed towards militarism and profiteering.

**Reply by sjeyakum** [reply]
Wow, I had never heard of that case before but it definitely highlights how programs are built by people who have biases, and how those biases can easily reveal themselves in the organization, performance, and functions of those programs. I also want to add on to your last comment about open-source efforts. While I understand that there is great importance to make sure data is publicly available and accessible, I also wonder if there is a limit to this, especially when the data includes sensitive information about vulnerable populations. How do you determine when the benefits of open-source efforts outweigh the costs of revealing information that may be perceived sensitive (i.e., health data, income, legal status, etc.) ? Should this be the only way we approach these cases (i.e., evaluating pros and cons)?

**Reply by kchhoeu** [reply]
I have never heard of WordNet database, that seems very focused on the classification of people that it really does not realize the importance of the nuances of each individual in a community. I feel that those type of systems bring in a lot more negatives in terms of how the data can be used. I think the push to keep cartographical tools inaccessible has to deal with the fact that a lot of the people in charge of these systems and how it is used do not come from a minority background, kind of like a way to silence those who do not participate directly.

**Comment by sjeyakum** [comment]
I once had to write an article about privacy and medical identity in telehealth services for a club. In medicine, privacy and security are very important as leaked patient health information can be exploited and used for inappropriate and dangerous purposes in the wrong hands. Connecting this back to data and GIS, I think one significant problem in this field is the challenge of protecting sensitive geographic data from cyberattacks and unauthorized access, especially since data breaches can have significant repercussions. For example, if organizations with geospatial data are targeted and attacked, hackers could potentially access sensitive information about critical infrastructure locations, emergency response plans, and vulnerable populations. Such situations could lead to severe consequences, which is why I think it's important to also work towards data security and protection as the field grows and GIS is used in more interdisciplinary ways.

**Reply by kchhoeu** [reply]
Your example about privacy and medical identity for telehealth services was really interesting! I never thought about how vulnerable health care services was both the programs and how it affects the patients. I think it is important like you mentioned to work towards data security and protection, in a way have the programs we are using to evolve to be used for more interdisciplinary ways and to protect the vulnerable whose data they have.

**Reply by Keiya-Wada** [reply]
I agree with you. Before reading your post, I was only thinking of things like the biases a dataset may have, so it was refreshing to see a completely new perspective. As you said, working towards strengthening privacy and security fits the field of GIS and the overall tech industry. Especially in such a sensitive field like medicine, I can definitely see where potential data leaks may cause a lot of issues.

**Reply by zhamic7** [reply]
I remember in this week's reading that someone made the argument that "GIS enabled American militarism in the Gulf War" (page 16). I agree that even if there may be problems with some datasets in creating accurate views of the world, data can ultimately reveal a lot of information about the source it was collected from. In the wrong hands or for the wrong purpose, this could have disastrous consequences. I suppose that this can also be another side to the problem of data collection -- not only do we want to ensure fewer biases, but we also want to ensure ethical collection (e.g., not taken from users without their consent, etc.).

**Reply by pwinnie2026** [reply]
As someone that does not intend to work in medicine or public health, it is interesting to hear a new perspective on this matter and how geographic data and ethics are intertwined when it comes to telehealth services. Especially as I know of multiple people using some form of telehealth services due to accessibility, cost, or etc. The integration of cybersecurity in medicine (while definitely not new) is something I am not as knowledgeable about and would be interested to learn more of! Even outside geographic data, looking at data breaches of 23andMe on users' geographic, financial, and genetic information could bring up concerns on how secure user data is and who can access this data now.

**Reply by sachigill** [reply]
I really enjoyed reading your response as this was definitely something I had not considered. As much as this data might be important for addressing issues regarding urban planning or public health, there are also major risks as data falling into the wrong hands can lead to serious outcomes, as you mentioned. As GIS becomes more intertwined with these fields, it becomes more important for there to be collaboration between GIS experts, cybersecurity specialists, as well as policymakers to go forth with strategies that can best leverage the advantages of GIS, while also mitigating the risks of data leaks or data falling into the wrong places.

**Reply by kimada62** [reply]
That was an interesting take that I would not have thought of. Most of my exposure to injustices relating to GIS is with biased mapping, but I didn't even think to consider how even fair and just mapping can pose a risk like that. I suppose having all the correct data still isn't enough to maintain mapping as a positive tool when there are so many security risks as well. I'd be interested to hear more.

**Comment by kchhoeu** [comment]
I think that the way data and GIS is used it is supposed to be concrete and objective, but the problems it causes it that the data can be overgeneralized or not used for the intended purposes. Because a lot of the research to collect the data and use is GIS is based on specific communities, their is a lot of biases, and just taking the information and using it to spread around without taking into account differences in the world rather than the data does more harm than good. Information from these could be misconstrued or classified data could also be leaked. I think because of these there is a lot of things to do to fix data and GIS to make it more safer and easier to access.

**Reply by Keiya-Wada** [reply]
I definitely agree with what you said. The main purpose of a dataset is to be objective but it fails to achieve that purpose with the way how we use datasets. I think it will be interesting to see how mapping will look like in the future, as it needs to address a ton of problems like the ones you mentioned. I think collecting data is also a very hard component, as you don't want to be too overgeneralized, but also not too specific.

**Comment by Keiya-Wada** [comment]
I think the problem with the way how we use GIS, similar to other technology, fails to capture the whole story of whatever it is that we are trying to address. One example of this with other tech is cameras. Similarly to GIS, it might serve as good evidence or data, but it is simply one frame, one shot of time taken out of the context of the whole story. Additionally, the person taking the picture can manipulate it to make it look like what they want. Although GIS does prove to be useful to some extent, it needs careful background check of where the data came from, who gathered the data, etc... By fully addressing the potential biases the map might be showing, that is when we can first look at the map and safely learn from it.

**Reply by zhamic7** [reply]
I also agree that oftentimes, data that people collect don't always reflect the whole picture. Not everyone is able to collect data on a large scale like corporations or government agencies like the U.S. Census Bureau are able to, and so oftentimes people end up relying on data collected by these organizations to conduct data analysis. This can lead to a misinterpretation of the results obtained since the means of data collection aren't always disclosed, so not everyone is made fully aware of why some data types were recorded while others weren't even collected in the first place.

**Reply by NellieNguyen149** [reply]
Hello Keiya! I fully agree with you, especially the part regarding needing background checks and being careful in general about where pictures, data, and information is sourced from. Calling a picture just one snapshot in time, and even then the picture can be changed to fit a certain narrative if someone desires to do so. It is important we stay cautious when interpreting data and making conclusions from it, and it would be good to examine and know what data sources are being used.

**Reply by pwinnie2026** [reply]
I totally agree with you! I can see this happening during elections in which polling data on various issue areas, candidates, etc. are portrayed differently or misinterpreted to portray a certain narrative. Especially as it becomes easier for online discourse and hearing other people's opinions, I agree that it is important to stay cautious and making direct conclusions even if people are more inclined to trust in data and statistics. Redlining originated from the U.S.'s government homeownership programs on "high" or "low" risk loans, further perpetuating historic injustices in communities of color because these metrics were often based on the racial demographics in these housing areas. Even if these agencies collected/utilizing data are “official” or government supported, it is important to recognize how (in cases like redlining) current power structures in place can lead to a distrust of data collection and resulting in gaps in the data.

**Reply by kimada62** [reply]
I agree and actually, this made me remember one map I was learning about in high school. The class was a Los Angeles history course that briefly discussed SoCal maps representing only select data on POC population relating to low income housing that ultimately perpetuated certain stereotypes. It made it difficult for ethnic groups to break out of marginalized and divided areas of housing in LA. It might've even mentioned the loans, but to be honest I'm unsure. In either regard, it is interesting how there is so much trust in this type of data. It leads to so much unconscious negligence and injustice.

**Comment by zhamic7** [comment]
Despite countless examples of AI failing to overcome the same biases our current power structure exhibits (e.g. facial recognition not being able to accurately identify non-white faces despite the widespread usage of such technology for security already), many people I know seem to believe that one day, with enough data and technical developments, these problems will just all go away by themselves. In other words, there is comparatively a considerable lack of attention being directed at data collection itself, and I believe that is the problem with our approach to using data for digital tools. We have somehow grown to wrongfully assume that our means of data collection are perfect and that the data we collect also perfectly reflect reality, and so the models that we use for analysis can thus present some truthful, universal story about the world. In actuality, the data one is able to collect is greatly limited by what roles and positions one holds in society, and technological tools like GIS can only create a picture of the world that oftentimes ends up ignoring the views of those people that aren't part of "us." This becomes increasingly problematic when the creators or users of some technology represent the dominant class as it only reinforces current power structures.

**Reply by kianaptran** [reply]
Hi Michelle, I agree entirely! There are so many unethical data collection practices that are used to exploit vulnerable populations such as those of low SES, children, etc. The issue of privacy is just one of the small problems where many users do not realize that their searches and other online habits are being tracked, once again making the user vulnerable to exploitation.

**Reply by NellieNguyen149** [reply]
Hello Michelle! I completely agree with you, especially the part about how the GIS doesn't always represent the full picture and who is included in the population represented ("us"). In my response, I also talked about how the group/population that we take our data from isn't always inclusive of everything/everyone, therefore the conclusions we make from the collected data are limited. People may continue to just accept current data collection methods without actively trying to become more inclusive and working toward diversifying and expanding our data collection sources and groups.

**Comment by NellieNguyen149** [comment]
I believe one of the biggest issues with the way we utilize data and GIS is that it can easily be skewed to favor a specific perspective or vision. It is extremely difficult to represent all viewpoints and avoid bias, especially because data collection is limited to the people who have access to tools where they can report information, opinions, or any type of data they can provide. Another factor that connects with limited data collection is the "silent" population that cannot communicate their voices due to simply not knowing how and where to report their ideas or statements (whether it's through surveys or other means). It seems that our natural instinct is to trust sources and information that have typical statistics and data because, at least for me, the more specific information is, the more 'plausible' it may be. However, it is crucial to review where our data is sourced from, but people do not always cross-reference one piece of information with multiple sources. In general, I feel that there should be a bigger emphasis on knowing not only what the data is, but where the data originated from, including who collected the data, who/what made up the population/subjects of which the data was collected from, and other identifiers that relate to the data.

**Reply by pwinnie2026** [reply]
I agree with you that there may be an assortment of reasons in which the "silent" population may not be equitably represented in the data. Especially in populations who are older, in communities with a language barrier, who may be working multiple jobs, or those who are nomadic or houseless. There are varying community conditions in which it may be harder for their voices to be accessed and reported. Even with surveys or the census, it is not enough for institutions and organizations to rely on digital communications or mailing addresses for responses. As you mentioned, meeting people where they are at, and having a bigger emphasis of identifiers that relate to the data is really important in both using and understanding the data/statistics in front of you.

**Comment by pwinnie2026** [comment]
One of the issues in how we use data and GIS comes with the relaxing of critical analysis, questioning, and bias-checking done when data and GIS are integrated into narratives. Data is not always collected, analyzed, or accessed equitably. In Crampton and Krygier's article, they discussed how even with GIS and data analytics tools becoming more readily available outside of GIS scientists and cartographers, "the distribution of these resources is spatially uneven" through the digital divide (Crampton & Krygier, 19). Therefore, when people share narratives with data, or encounter some form of statistics or map, it is not enough to always accept it as it is integrated into whatever narrative it is being used in. Maps, graphs, and other forms of data/GIS visualization are important in conveying information readily accessible, especially in populations with varied literacy and education levels. Whilst it can be more accessible to understand than academic journals, it is important for us to recenter how the data was collected, where the faults in the data lie, and who/what is being represented in the data and who/what is being left out from this data. For Native Hawaiian and Pacific Islander communities, hearing community leaders at UCLA's AANHPI Policy Summit present about the community conditions impacting their communities' housing insecurity and overcrowding is an example of the importance of uplifting their community narratives in accompaniment with the data.  Especially with the current power structures approaching issues with a problem-solution mindset, not every organization or federal bureau will expend their resources and time towards this approach leading to these "gaps" and "silences".

**Reply by nesslee123** [reply]
Really great response! I completely agree with you because it has become commonplace to accept information without checking for accuracy and bias. I also agree that maps and graphs often help manipulate viewers into believing data even though this information may not fully or accurately encompass the topic being discussed or represented. This is a big issue today because people have begun accepting information being portrayed online or in the media without questing the issues for accuracy.

**Reply by caitlynn-hauw** [reply]
Wow! This was really insightful! Although Crampton and Krygier did briefly touch on the very-real digital divide, they might have actually over-emphasized how accessible map-making truly is; I believe they mentioned this point maybe three times. I believe there can be a high barrier-to-entry to map making that makes it difficult for many people who lack access to education or wealth to participate in mapmaking. I really appreciate you bringing up this point, because I think it's a lesser talked about inequity within cartography, especially when we're striving for equitable, participatory cartography.

**Comment by nesslee123** [comment]
One problem with the way we use data and GIS is that data is often subjective. Although data may seem objective, it is often fueled by political agendas which may manipulate or sway people’s beliefs to align with the data. Crampton & Krygier (2010) state, “Cartography’s latest ‘technological transition is not so much a question of new mapping software but a mixture of ‘open source’ collaborative tools, mobile mapping applications, and geotagging. While this trend has been apparent to industry insiders for some time, a more social theoretic critique, which we argue is a political one, situates maps within specific relations of power and not as neutral scientific documents” (p. 12). The article emphasizes the influence of biased data on the public and how data may align with political agendas since it remains subjective.

**Reply by sachigill** [reply]
I definitely agree with how data may push political agendas or attempt to swing beliefs to align with it. This reminds of the concept of gerrymandering. When political parties draw the boundaries of their electoral districts in order to favor a certain political party or ethnic group, it creates an unfair election but also skews the data and public's perceptions, as it might seem someone may have won “fairly” or is the popular candidate when in reality that's not the case, since the people and their voices were not being fairly represented.

**Comment by sachigill** [comment]
I believe a huge issue when it comes to data and GIS is bias, since a lot of the times people collecting or manipulating the data come from groups in power, meaning we only get to see a one sided perspective. The perspectives from those whose voices have been traditionally silenced or from populations who may not have access to these technological tools and knowledge, tend to be overlooked. One example being how there are many maps out there that label certain places with colonizer names and not the native name. However, according to Crampton and Krygier, the rise of mobile and other mapping techniques have democratized map making, helping to shift the power from experts to the general public, bridging some of the existing disparities.

**Reply by caitlynn-hauw** [reply]
Your response reminds me of Crampton and Krygier's quotation of Yves Lacoste, “The map has fundamentally been an instrument of power" (1973: 1). I thought it was interesting how Crampton and Krygier explain the historical context behind maps and how the use of maps in World War II by the Axis and Allied powers drove Robinson to seek an "unbiased and reliable map." However, as you mentioned, even an "unbiased and reliable map" is subject to the biases of the person creating it, and oftentimes, this was people, specifically white men, in positions of power. I don't believe you'll answer, but I wonder what you would think would be more harmful: a map that is overtly biased or a map that feigns objectivity.

**Comment by kimada62** [comment]
I remember taking a climate change course that discussed the rising sea levels in New Orleans and how maps representing areas for mandatory flood insurance was biased to social approval. So, there are houses that should probably purchase insurance, but with the complaints over the city's high flood insurance, the maps were changed. With the looming question of if these maps were accurate, homeowners were at high risk of being left uninsured with a flooded house after the next disastrous event. GIS and data has a critical power to tell the story of an area and help cities and homeowners take the necessary steps to help fund, support, and bring together communities. However, social pressures and political agendas often lead to only certain data being shared, allowing mapmakers to tell a subjective story on what would be assumed to be an objective document.

**Reply by kimada62** [reply]
Found one of the old articles I had to read, if there's interest in what I'm talking about with New Orleans:

https://www.npr.org/2016/09/30/495794999/new-maps-label-much-of-new-orleans-out-of-flood-hazard-area

**Reply by albertkun** [reply]
awesome, thanks for sharing!

**Comment by caitlynn-hauw** [comment]
The way in which we collect data is described as methodical, objective, and scientific. But who collects that data? And on whom? Who decides what matters? Who has the power? The seemingly unbiased data collection process is tainted by a power imbalance wielded by the data collector that makes the process impossibly objective. Similarly, with maps, even the idea of what is a map is fixed: there are no Greenland Inuit coastal charts carved from wood or Marshall Island stick charts, but a Mercator projection on paper. Even the connotation of the language used to describe maps can affect the perceived credibility, like artistic and subversive versus unbiased and reliable. But who determined what a map was and who determined what a good one should be? Our use of data and GIS consists of contrived binaries—like art and science, objective and subjective, and scientific and ideological—that mislead us into believing that our western-way of thinking is "correct." As Crampton & Krygier state, maps are inherently political. The maps we create are capable of demolishing villages, wiping out cities, and destroying whole worlds; it can cement who has power and who doesn't by creating models on pieces of papers that will be used for centuries to uphold power structures and imbalances.
